DynamicTablesPkg/Include/Library/AmlLib/AmlLib.h

```diff
@@ -374,7 +374,7 @@ EFI_STATUS
 EFIAPI
 AmlDeviceOpUpdateName (

```

Comment:
Is it possible to check the call sites of the updated functions ?
I think unnecessary casts can be removed with this patch e.g.:
`  Status = AmlDeviceOpUpdateName (DeviceNode, (CHAR8 *)Name);`
->
`  Status = AmlDeviceOpUpdateName (DeviceNode, Name);`

Reply:
I've updated this PR with another commit that removes the two instances of casts that I was able to find in the edk2 repo.